app/models/shipit/stack.rb

```diff
@@ -182,6 +182,12 @@ def deployed_too_recently?
     end
 
     def async_refresh_deployed_revision
+      async_refresh_deployed_revision!
+    rescue => error
+      logger.warn "Failed to dispatch FetchDeployedRevisonJob: #{error.message}"
```

Comment:
Would be good to include the class name here: `[#{error.class.name}] #{error.message}`